pkg/render/apiserver.go

```diff
@@ -679,7 +679,7 @@ func (c *apiServerComponent) apiServerService() *corev1.Service {
 					Name:       "apiserver",
 					Port:       443,
 					Protocol:   corev1.ProtocolTCP,
-					TargetPort: intstr.FromInt(apiServerPort),
+					TargetPort: intstr.FromInt(ApiServerPort),
```

Comment:
It gets referenced in the Tiers render component since policy for API server is currently there

I did some after-the-fact splitting of commits to make review easier, so if you are reviewing commit-by-commit perhaps you encountered an imperfection of that split, apologies if so